script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -355,7 +355,7 @@ If the thieves who rummaged through my library failed to heed that advice, then
 [sha:90c1dcfda6977a7bf0d83c709763e023dc3847f4]{
 -- Page 99, Offset 30821.
 -- 「残念ながら。私の研究は魂の分野に切り替わった。
-"Tis unfortunate. My research changed to focus primarily on the soul.
+"'Tis unfortunate. My research changed to focus primarily on the soul.
```

Comment:
revert

Reply:
you sure? I never saw an instance without '

Reply:
'Tis is more standard.